src/main/scala/edu/colorado/plv/cuanto/scoot/ir/BinopExpr.scala

```diff
@@ -0,0 +1,18 @@
+package edu.colorado.plv.cuanto.scoot.ir
+
+/**
+  * Created by Jared on 6/13/2017.
+  */
+abstract class BinopExpr(dt: soot.jimple.BinopExpr) extends Expr(dt: soot.jimple.Expr) {
```

Comment:
Per our discussion, this would be come a trait (with no arguments). Just a type name.